settings.gradle

```diff
@@ -42,6 +42,10 @@ include ":opentelemetry-all",
         ":opentelemetry-sdk-contrib-jaeger-remote-sampler",
         ":opentelemetry-bom"
 
+if(JavaVersion.current().isJava11Compatible()) {
```

Comment:
For some reason this doesn't seem to work with CricleCI, I don't see the JFR project showing up. Running gradle locally with JDK 8 and JDK 11 it is ignore and added correctly